wallet/dapps-communication.md

```diff
@@ -612,12 +612,16 @@ properties:
   amount:
     type: number
     description: Set the amount of MASSA coins given to the block creator.
+  dryRun:
+    type: boolean
+    description: If true, the transaction will run in dry-run mode so that the result can be checked without actually broadcasting the transaction.
```

Comment:
The fact that the transaction is not broadcasted is not the main point here: the main purpose is that the modification done to the ledger by the execution of the smart contract is not persisted.